deps/GraphBLAS/.gitignore

```diff
@@ -1,69 +0,0 @@
-# Ignore these files:
```

Comment:
Let's bring this file back, and add as much as we can to it. I think everything within `Source/Generated/` can be safely ignored, right?